ci-operator/config/openshift/release/openshift-release-master__nightly-4.18.yaml

```diff
@@ -1167,6 +1167,7 @@ tests:
       PACKET_OS: rocky_9
       SKEW: 90d
     workflow: openshift-e2e-cert-rotation-shutdown-sno
+  timeout: 6h0m0s
```

Comment:
The `timeout` is only applied to 3y and above?

Reply:
Yes, others fit in standard 4h timeout